app/pods/components/search-attempt/component.js

```diff
@@ -0,0 +1,58 @@
+import Component from "@ember/component";
+import { later } from "@ember/runloop";
+import { task } from "ember-concurrency";
+import { action } from "ember-decorators/object";
+import { alias } from "ember-decorators/object/computed";
+import { service } from "ember-decorators/service";
+
+export default class SearchComponent extends Component {
+  tagName = 'span'
+  hideResultsBox = true;
+  qs = '';
+  @alias('searchTask.lastSuccessful.value') response
+  
+  @service router
+  @service store
+  
+  searchTask = task(function* () {
+    const searchQuery = yield this.get('qs')
+    const contents = this.get('store').peekAll('content');
```

Comment:
We only want to search this contents in this runAttempt

Reply:
Done @abhishek97